_posts/2018-12-27-dropping-puppet4.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Dropping Puppet 4 support
+date: 2018-12-27
+github_username: bastelfreak
+---
+
+Puppet 4 will be end of life by the December 31 2018. There isn't an official
+support matrix for the FOSS components. For Puppet enterprise this can be found
+[here](https://puppet.com/misc/puppet-enterprise-lifecycle). PE 2016.4 is the
+oldest currently support version. It currently ships
+[Puppet 4](https://puppet.com/docs/pe/2017.3/component_versions_in_recent_pe_releases.html).
+
+By January 1 2019, Puppet Enterprise 2018.1 will be the oldest supported
+version by Puppet Inc. It contains Puppet 5.5.8. Vox Pupuli will remove Puppet
+4.x from the metadata.json and from the Travis CI testmatrix in January. Our
```

Comment:
testmatrix

Reply:
hey, I'm not quite sure what you mean. Is there a typo I cannot see?